src/common-column-defs.ts

```diff
@@ -60,31 +60,31 @@ export const columnDefs: IColumnDef[] = [
     pluginType: PluginType.WindowShade,
     pluginSubType: WindowShadeType.TeacherTip,
     eventMatcher: /TeacherEdition-windowShade-TeacherTip Tab(Opened|Closed)/,
-    eventSubType: undefined
+    eventSubType: EventSubType.WindowShadeTeacherTip
   },
   {
     title: "Window Shade - Theory & Background",
     shortTitle: "WS-TB",
     pluginType: PluginType.WindowShade,
     pluginSubType: WindowShadeType.TheoryAndBackground,
     eventMatcher: /TeacherEdition-windowShade-TheoryAndBackground Tab(Opened|Closed)/,
```

Comment:
From what I remember in the logs it seems like this event matcher should actually be:
`/TeacherEdition-windowShade-TeacherTip Tab(Opened|Closed)/`

The text after the second dash comes from the action:
https://github.com/concord-consortium/teacher-edition-tips-plugin/blob/0a07ca3878a8d1bf7f3f1f7b21ce391799257e00/src/components/window-shade.tsx#L89

Which is globally defined here:
https://github.com/concord-consortium/teacher-edition-tips-plugin/blob/0a07ca3878a8d1bf7f3f1f7b21ce391799257e00/src/utilities/analytics.ts#L13

So the action is always going to be either:
`TeacherTip TabOpened` or `TeacherTip TabClosed`

In case we decide to fix this in the future and make it less confusing, I'd suggest changing this pattern to:
`/TeacherEdition-windowShade-.*Tab(Opened|Closed)/`